hm_1/tasks/task_02_05.py

```diff
@@ -7,15 +7,35 @@ def detect_index(some_iterable_object, position):
     return some_iterable_object.index(position)
 
 
-def custom_range(some_iterable_object, start="", stop="", step=1):
+def detect_stop_start_step(some_iterable_object, args):
+    if not args:
+        args = 0, len(some_iterable_object), 1
+        return args
```

Comment:
Keep it simple. Same for other code.
```suggestion
        return 0, len(some_iterable_object), 1
```